apps/music/js/metadata/ogg.js

```diff
@@ -116,14 +172,22 @@ var OggMetadata = (function() {
    * @param {BlobView} page The audio file being parsed.
    * @return {Object} An object containing the page's segment table.
    */
-  function readPageHeader(page) {
+  VorbisCommentParser.prototype.readPageHeader = function(page) {
     var capture_pattern = page.readASCIIText(4);
     if (capture_pattern !== 'OggS') {
       throw new Error('malformed ogg page header');
     }
 
+    /*var version =*/ page.readUnsignedByte();
+    /*var type =*/ page.readUnsignedByte();
+
+    page.advance(8); // granule
+
+    /*var serial =*/ page.readUnsignedInt();
+    /*var seq =*/ page.readUnsignedInt();
+
```

Comment:
ah yeah. I actually wanted to check the serial and sequence for reading multiple pages. Sounds like I forgot.